### ISSUE TITLE
AI generates black images

### ISSUE THREAD

**Issue by TheTomyLV** [mallorbc/stable-diffusion-klms-gui#8]
Any image I want to generate returns a black box, i tried changing all the settings in the gui and all that, and the container seems to be working fine and no errors
![Screenshot 2022-09-08 182244](https://user-images.githubusercontent.com/62905605/189162661-a314cf48-bb08-4f29-bc7f-5c2b2b402de5.png)
![Screenshot 2022-09-08 182317](https://user-images.githubusercontent.com/62905605/189162673-1632f2d9-0aa7-40c2-b921-b6b2861c929a.png)

**Comment by TheTomyLV** [comment]
Running windows wsl2

**Comment by Mallku2** [comment]
I'm getting the same behavior on Arch Linux, with a NVIDIA GeForce GTX 1660 Ti, 6GB VRAM, using the last version of the main branch.

**Comment by rgriffogoes** [comment]
I'm far from an expert, but I read somewhere that the model works well with 512x512 dimensions. Going lower resolution will misbehave. 

Did you try higher resolutions? I'm assume it would crash without it, but did you download the model/weights and placed in the right folder? (./diffusion_model/model.ckpt)

**Comment by Mallku2** [comment]
@rgriffogoes Thanks a lot! Indeed, I needed to reduce the dimensions (less than 512x512), because, if not, the process crashed with an "out of memory" error.

With regard to the model/weights, I followed the tutorial mentioned in the repo of the project: https://www.youtube.com/watch?v=6x87ah70rcY. Also, thanks for pointing this out.

Well, I guess that this means no stable diffusion for me, for the moment xD
